src/RBCXOled.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#include "WString.h"
+
+namespace rb {
+
+/**
+ * \brief Helper class for controlling the Oled.
+ */
+class Oled {
+    friend class Manager;
+
+public:
+    typedef enum {
+        Oled_128x32,       
+        Oled_128x64,       
+    } OledType;
+
+    // Enumeration for screen colors
+    typedef enum OledColor{
+        Black = 0, // Black color, no pixel
+        White = 1  // Pixel is set. Color depends on OLED
+    } OledColor;
+
+    typedef enum {
+        Font_6x8,
+        Font_7x10,
+        Font_11x18,
+        Font_16x26,        
+    } OledFontDef;
+
+
+    void init(OledType type, bool rotate = true, bool inverseColor = false);
+    void fill(OledColor color = White);
+    void updateScreen(void);
+    void drawPixel(uint8_t x, uint8_t y, OledColor color = White);
+    void writeString(const char* str, OledFontDef Font, OledColor color = White);
+    void writeString(char* str, OledFontDef Font, OledColor color = White);
+    void writeString(String str, OledFontDef Font, OledColor color = White);
```

Comment:
Stačí `writeString(const char *str, ..` a k tomu `writeString(const std::string& str, ..` který jen provolá tu první funkci, `writeString(str.c_str(), Font, color);`., není třeba mít 3x to stejné.